src/editors/containers/VideoEditor/components/VideoSettingsModal/components/VideoSourceWidget/index.jsx

```diff
@@ -98,7 +98,7 @@ const VideoSourceWidget = () => {
         <FormattedMessage {...messages.fallbackVideoMessage} />
       </div>
       {fallbackVideos.formValue.length > 0 ? fallbackVideos.formValue.map((videoUrl, index) => (
-        <Form.Row className="mt-3.5 mx-0 flex-nowrap">
+        <Form.Row className="mt-3.5 mx-0 flex-nowrap" key={videoUrl}>
```

Comment:
We need something like the following:
```suggestion
        // eslint-disable-next-line react/no-array-index-key
        <Form.Row className="mt-3.5 mx-0 flex-nowrap" key={`${index}-${videoUrl}`}>
```
Because otherwise there's a very weird bug when the URLs are empty or the same:


https://github.com/user-attachments/assets/b85160f9-7867-4702-b084-b922cbc845fe